workflows/n_t.snake

```diff
@@ -125,26 +140,28 @@ localrules:
     gone_clone,
     gone_copy,
     gone_params,
-    all_plot
 
 
 rule all:
-   input: 
-    expand(output_dir + "/plots/{demog}/{chrms}/{dfes}/{annots}/all_estimated_Ne.pdf", 
-        chrms=chrm_list, 
-        demog=demo_model_ids, 
-        dfes=dfe_list, 
-        annots=annotation_list,
-        seeds=seed_array,
-        ),
-    rules.simulation_all.output,
+    input:
+        #rules.simulation_all.output,
+        expand(output_dir + "/plots/{demog}/estimated_Ne_t_final.csv", demog=demo_model_ids),
+        expand(output_dir + "/plots/{demog}/estimated_Ne_t_final.pdf", demog=demo_model_ids),
+        expand(output_dir + "/plots/{demog}/coal_estimated_Ne_t.csv", demog=demo_model_ids),
+        expand(output_dir + "/plots/{demog}/{method}/{method}_estimated_Ne_t.csv",
+            demog=demo_model_ids, method=methods),
+        expand(output_dir + "/plots/{demog}/{method}/{method}_estimated_Ne_t.pdf",
+            demog=demo_model_ids, method=methods),
```

Comment:
```suggestion
    input:
        #rules.simulation_all.output,
        expand(output_dir + "/plots/{demog}/{method}/{method}_estimated_Ne_t.pdf",
            demog=demo_model_ids, method=methods),
```

Reply:
I can see how that would work since the pdf is dependent on the csv. However, we need to keep the second line ... "/plots/{demog}/estimated_Ne_t_final.pdf" since that is the final plot. Also coal_estimated_Ne_t.csv since no other rules (see rule graph) generate this.

Reply:
I meant to leave the final output, estimated_Ne_t_final.pdf, uncommented. Sorry.

I think it would seem cleaner to make the individual rules require coal_estimated_Ne_t.csv as input. But we don't have to conform to my way, just a suggestion.